command/server/generate_config.go

```diff
@@ -0,0 +1,45 @@
+package server
+
+import (
+	"fmt"
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v3"
+	"os"
+)
+
+func getGenerateConfigCmd() *cobra.Command {
+	configCmd := &cobra.Command{
+		Use:   "export",
+		Short: "export default-config.yaml file with default parameters that can be used to run the server",
+		Run:   runGenerateConfigCommand,
+	}
+
+	return configCmd
+}
+
+func runGenerateConfigCommand(cmd *cobra.Command, _ []string) {
```

Comment:
What do you think about adding some kind of export success message?

See command results for other commands on how we did this

Reply:
Added success output message 9350d7d3c963f0b39070691337bd8610bbf3da8a